sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/eventhubs/properties/AzureEventHubsProperties.java

```diff
@@ -108,7 +108,7 @@ public Processor buildProcessorProperties() {
         propertyMapper.from(this.processor.getConsumerGroup()).to(properties::setConsumerGroup);
 
         propertyMapper.from(this.processor.trackLastEnqueuedEventProperties).to(properties::setTrackLastEnqueuedEventProperties);
-        propertyMapper.from(this.processor.initialPartitionEventPosition).to(properties::setInitialPartitionEventPosition);
+        BeanUtils.copyProperties(processor.initialPartitionEventPosition, properties.getInitialPartitionEventPosition());
```

Comment:
please don't use this, we should stick to the propertyMapper. propertyMapper.from(this.processor.initialPartitionEventPosition).when().to(xxx -> xxx)

Reply:
can we also add some hints for this?

Reply:
I can not find a good way to add a hit for this map property, the same with `authorizationClients` in AAD properties.